app/src/main/java/nerd/tuxmobil/fahrplan/congress/sharing/models/SessionExport.kt

```diff
@@ -33,10 +33,11 @@ data class SessionExport(
             url = session.url,
             speakers = session.speakers.joinToString(";"),
             track = session.track,
-            type = session.lang,
-            lang = session.abstractt,
-            abstract = session.description,
-            description = session.links,
-            links = session.date
+            type = session.type,
+            lang = session.lang,
+            abstract = session.abstractt,
+            description = session.description,
+            links = session.links,
+            date = session.date
```

Comment:
Do we need `date` at all? Chaosflix does not care as far as I know. Sadly, I cannot verify it either. It seems the import function in Chaosflix no longer works on recent Android version.

If you think a date/time information should be there I recommend picking `dateUtc` instead like this:

``` kotlin
date = Moment.ofEpochMilli(session.dateUTC).toString()
```

Reply:
Oh, ok. You suggested more changes.

I believe there shouldn't be any harm in having the date, if it doesn't break compatibility with the originally consuming app. I'd appreciate having access to it.

> Since the field has never been exported I would even rename it to `startsAt` resp. `starts_at` for the JSON export which is much clearer than `date`.

I am not quite sure where you want that inserted. The suggestion sounds good, I just don't know how to implemented it, as I have never used Kotlin before. Maybe changes at that level would be better done by someone who actually knows what they are doing. :rofl:

Reply:
> I'd appreciate having access to it.

OK, then lets put it in.

It's not too hard. You basically have to replace this line:

``` kotlin
var date: String? = null
```

with:

``` kotlin
@Json(name = "starts_at")
var startsAt: String? = null,
```

and:

``` kotlin
date = session.date
```

with:

``` kotlin
startsAt = Moment.ofEpochMilli(session.dateUTC).toString()
```

Reply:
Ok, needed to figure out how to do imports, tests have passed, hope its the way you wanted it.